### PR TITLE
Fix skip_if_open marker use

### PR DESCRIPTION
**Pull request by mshriver**
I broke this in #8092, missed in review/CI.

### CODE REVIEW

**Review by JacobCallahan — APPROVED** [review]
ACK

File: tests/foreman/virtwho/cli/test_kubevirt.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def virtwho_config(form_data):
     return VirtWhoConfig.create(form_data)['general-information']
 
 
-@pytest.mark.skip_if_open('BZ:1735540', reason='We have not supported kubevirt hypervisor yet')
+@pytest.mark.skip_if_open('BZ:1735540')
```
Comment (hkx303) [inline tests/foreman/virtwho/cli/test_kubevirt.py:54]: Thanks Mike, but I prefer to use the `skipif` to keep it in line with https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/virtwho/api/test_kubevirt.py#L58
```
@pytest.mark.skipif(
    condition=(is_open('BZ:1735540')),
    reason='We have not supported kubevirt hypervisor yet',
)
```

Comment (hkx303) [reply tests/foreman/virtwho/cli/test_kubevirt.py:54]: Or we should also modify this in https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/virtwho/api/test_kubevirt.py#L58

Comment (mshriver) [reply tests/foreman/virtwho/cli/test_kubevirt.py:54]: I would actually prefer to use skip_if_open, as BZ skipping is the specific use for this marker. I can update api/test_kubevirt to use it as well?  Going this direction also means we don't have to import the is_open method into test modules.

Comment (mshriver) [reply tests/foreman/virtwho/cli/test_kubevirt.py:54]: @hkx303 commit updated with both uses being consistent with `skip_if_open`, thanks for being flexible!

**Review by lpramuk — APPROVED** [review]

**Review by hkx303 — COMMENTED** [review]
Then we should also delete this line :)
https://github.com/SatelliteQE/robottelo/blob/49f47af7d01094b740839aef14ec28548ad1ed83/tests/foreman/virtwho/api/test_kubevirt.py#L23

**Review by hkx303 — APPROVED** [review]
Give my ACK as I hope we can merge it today (US time), thank you!

**Review by hkx303 — APPROVED** [review]

**Pull request merged**